website_search.py

```diff
@@ -0,0 +1,36 @@
+import requests
+from bs4 import BeautifulSoup
+import json
+
+url = input("[+} URL : ")
+
+raw_code = requests.get(url).text
+
+# Beautiful : Translate text to html
+soup = BeautifulSoup(raw_code, "lxml")
+
+data_dictionary = dict()
+
+try:
+    website = soup.select(".biz-website > a")[0].string
+    data_dictionary["website"] = website
+except Exception as ex: 
+    data_dictionary["website"] = 'unknown'
+
+moreinfo_dictionary = dict()
+
+moreinfo = soup.select(".short-def-list > dl")
+for info in moreinfo:
+    attr_name, attr_content = "", ""
+    for  data in info.children:
```

Comment:
remove extra space after 'for'

Reply:
Wow! Your review skill is awesome,,, I envy you! Thank you